### DISCUSSION TITLE
cannot upgrade to libImath-3_1.30?

### DISCUSSION THREAD

**Discussion by uwaovo** [Homebrew/discussions#4867] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: a8519f78fb63f2f2266950bdd8141037da69f8bd
Last commit: 5 weeks ago
Core tap JSON: 01 Oct 13:30 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_NO_AUTO_UPDATE: set
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 14.0-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: 15.0
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

So I met problem in gdal program like:
```zsh
dyld[73991]: Library not loaded: /opt/homebrew/opt/imath/lib/libImath-3_1.30.dylib
  Referenced from: <8C8CF8DC-9B0D-3688-AEF8-CB4C51C2AE2D> /opt/homebrew/Cellar/gdal/3.6.4_4/lib/libgdal.32.3.6.4.dylib
```
I checked the path and just only found: `libImath-3_1.29.8.0.dylib`. I also tried `brew upgrade imath` and got:
```zsh
Warning: imath 3.1.9 already installed
```
Any suggestions 👀?

**Comment by SMillerDev** [comment]
Does the file exist?

> Core tap JSON: 01 Oct 13:30 UTC

That's also a very long time ago. Do you have automatic updates disabled ?

**Reply by uwaovo** [reply]
Thanks Reply. I have now turned on automatic updates and displayed `Core tap JSON: 28 Oct 01:59 UTC` in brew config. 
This also did not solve the problem. The file of `libImath-3_1.30.dylib` does not exist, I got the latest `libImath-3_1.29.8.0.dylib`.
I guess brew didn't update imath to the latest compiled version? Version 3_1.30 can be compiled from the latest imath repository.
But you know, I prefer to use brew.

**Reply by SMillerDev** [reply]
What does `brew outdated` say?

**Comment by uwaovo** [comment]
Use vgpkg to install imath so that there will be no exceptions.
